src/main/java/com/google/api/gax/core/FlowControlSettings.java

```diff
@@ -49,12 +49,23 @@ public static FlowControlSettings getDefaultInstance() {
   @Nullable
   public abstract Integer getMaxOutstandingRequestBytes();
 
+  /** Is flow control enabled. Defaults to true. */
+  public abstract boolean getIsEnabled();
+
+  /**
+   * The behavior of FlowController when the specified limits are exceeded. Defaults to
+   * ThrowException.
+   */
+  public abstract LimitExceededBehavior getLimitExceededBehavior();
+
   public Builder toBuilder() {
     return new AutoValue_FlowControlSettings.Builder(this);
   }
 
   public static Builder newBuilder() {
-    return new AutoValue_FlowControlSettings.Builder();
+    return new AutoValue_FlowControlSettings.Builder()
+        .setIsEnabled(true)
+        .setLimitExceededBehavior(LimitExceededBehavior.ThrowException);
```

Comment:
It occurs to me that another option is to have a third enum value, `Ignore`, which we could use instead of the "enabled" flag.